modules/proc_use-quay-build-dockerfiles.adoc

```diff
@@ -204,43 +207,31 @@ Use the following procedure to create a Build trigger.
+. Enter a repository name, for example, `\https://github.com/<username>/testrepo`. Then, click *Next*. 
 
 . When prompted, configure the tagging options by selecting one of, or both of, the following options:
 +
 * *Tag manifest with the branch or tag name*. When selecting this option, the built manifest the name of the branch or tag for the git commit are tagged.
 +
 * *Add `latest` tag if on default branch*. When selecting this option, the built manifest with latest if the build occurred on the default branch for the repository are tagged. 
++
+Optionally, you can add a custom tagging template. There are multiple tag templates that you can enter here, including using short SHA IDs, timestamps, author names, committer, and branch names from the commit as tags. For more information, see "Tag naming for Build triggers".
++
+After you have configured tagging, click *Next*.
 
-. Optional. Add a custom tagging template. There are multiple tag templates that you can enter here, including using short SHA IDs, timestamps, author names, committer, and branch names from the commit as tags. For more information, see "Understanding tag naming for Build triggers".
-
-. Click *Continue*.
-
-. When prompted, select the location of the Dockerfile to be built when the trigger is invoked. If the Dockerfile is located at the root of the git repository and named Dockerfile, enter */Dockerfile* as the Dockerfile path.
-
-. Click *Continue*.
+. When prompted, select the location of the Dockerfile to be built when the trigger is invoked. If the Dockerfile is located at the root of the git repository and named Dockerfile, enter */Dockerfile* as the Dockerfile path. Then, click *Next*. 
 
-. When prompted, select the context for the Docker build. If the Dockerfile is located at the root of the Git repository, enter `/` as the build context directory.
+. When prompted, select the context for the Docker build. If the Dockerfile is located at the root of the Git repository, enter `/` as the build context directory. Then, click *Next*. 
 
-. Check for any verification warnings. If necessary, fix the issues before clicking *Continue*. 
+. Optional. Choose an optional robot account. This allows you to pull a private base image during the build process. If you know that a private base image is not used, you can skip this step. 
 
-. When prompted with *Ready to go!*, click *Continue*. You are redirected to a confirmation page.
+. Click *Next*. Check for any verification warnings. If necessary, fix the issues before clicking *Finish*. 
 
-. Save the SSH Public Key, then click *Return to <organization_name>/<repository_name>*. You are redirected to the *Builds* page of your repository. 
+. You are alerted that the trigger has been successfully activated. Note that using this trigger requires the following actions:
+* You must give the following public key read access to the git repository.
+* You must set your repository to `POST` to the following URL to trigger a build.
```

Comment:
Depending on the line 231 and 232,  I guess you are taking `custom Git repository` as example.  
For `github repository push` trigger, customer doesn't need to do these,  quay has done these steps in the back-end with the credentials in config.yaml.

Reply:
I am going to use a custom Git repo for this example :) I have changed some wording to reflect that.